content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -102,18 +102,16 @@ By default, a waypoint will only handle traffic destined for **services** in its
 
 It is also possible for the waypoint to handle all traffic, only handle traffic sent directly to **workloads** (pods or VMs) in the cluster, or no traffic at all. The types of traffic that will be redirected to the waypoint are determined by the `istio.io/waypoint-for` label on the `Gateway` object.
 
-The `--for` parameter to `istioctl waypoint apply` can be used to change the [traffic type](#waypoint-traffic-types) redirected to the waypoint:
+Use the `--for` argument to `istioctl waypoint apply` to change the types of traffic that can be redirected to the waypoint:
 
-| `waypoint-for` value | Traffic type |
+| `waypoint-for` value | Original destination type |
 | -------------------- | ------------ |
 | `service`            | Kubernetes services |
 | `workload`           | Pod IPs or VM IPs |
 | `all`                | Both service and workload traffic |
 | `none`               | No traffic (useful for testing) |
 
-{{< tip >}}
-Traffic type concerns the original addressing. To-service traffic does not become to-workload once the service is resolved to a pod by the service mesh.
-{{< /tip >}}
+Waypoint selection occurs based on the type of destination, `service` or `workload`, which traffic was _originally addressed to_. If traffic is addressed to a service which does not have a waypoint, a waypoint will not be transited: even if the eventual workload it reaches _does_ have an attached waypoint.
```

Comment:
```suggestion
Waypoint selection occurs based on the type of destination, service, or workload, which traffic was _originally addressed to_. If traffic is addressed to a service which does not have a waypoint, a waypoint will not be transited: even if the eventual workload it reaches _does_ have an attached waypoint.
```

Reply:
Oxford comma?

Also, I had added the backticks here to highlight that `service` and `workload` are configurable settings not just generic concepts. Maybe that is incorrect still.

Reply:
Yes, we do oxford commas. And gotcha, I guess I didn't read it that way, but I guess that's what I get for only reading the change and not the text above. I've modified my suggestion that might clarify that a bit more

Reply:
OK, took another crack and making this read a little cleaner based on, but not exactly adopting your suggestion